src/components/myButton/index.tsx

```diff
@@ -0,0 +1,18 @@
+import { ReactNode } from "react";
+
+type Props = {
+  onClick: () => void;
+  children?: ReactNode;
+  buttonWidth: number;
+};
+export const MyButton = ({ onClick, buttonWidth, children }: Props) => {
```

Comment:
well, MyButton name looks cute, but use something more readable, like BaseButton or PrimaryButton etc. 
Same for MySearching